### PR TITLE
clarification: VehicleStopStatus values

### PR DESCRIPTION
**Pull request by scmcca**
As discussed in #291. (cc: @sven4all)

### CODE REVIEW

**Review by barbeau — APPROVED** [review]

**Comment by barbeau** [comment]
+1 for USF

**Comment by scmcca** [comment]
@sven4all Good point, there is some redundancy. We can simply take out a portion of the phrase so it reads:

> The vehicle ~~has departed and~~ is in transit to the referenced stop.

IMHO "in transit" is more intuitive than "en route". Thoughts?

**Comment by sven4all** [comment]
In transit to is also fine for me.

**Comment by e-lo** [comment]
LGTM

File: gtfs-realtime/spec/en/reference.md [hunk 1]
```diff
@@ -259,9 +259,9 @@ Realtime positioning information for a given vehicle.
 
 | _**Value**_ | _**Comment**_ |
 |-------------|---------------|
-| **INCOMING_AT** | The vehicle is just about to arrive at the stop (on a stop display, the vehicle symbol typically flashes). |
-| **STOPPED_AT** | The vehicle is standing at the stop. |
-| **IN_TRANSIT_TO** | The vehicle has departed the previous stop and is in transit. |
+| **INCOMING_AT** | The vehicle is just about to arrive at the referenced stop (on a stop display, the vehicle symbol typically flashes). |
+| **STOPPED_AT** | The vehicle is standing at the referenced stop. |
+| **IN_TRANSIT_TO** | The vehicle is in transit to the referenced stop. |
```
Comment (barbeau) [inline gtfs-realtime/spec/en/reference.md:264]: We should mark `IN_TRANSIT_TO` as the default value here too in the reference.

**Comment by barbeau** [comment]
While we're here, looking back at #291, to really fix the ambiguity discussed there I think we should be more explicit about the expected behavior for `current_status` in coordination with `current_stop_sequence`.

What do you think about adding something like the below explanation for how current_stop_sequence should be updated?

>If `current_status` is provided, current_stop_sequence should be updated to the next stop after the bus departs from the previous stop.
>
>Specifically, this should be the sequence of the vehicle going from stop_sequence 2 to 3 if both fields are provided:
>
>current_stop_sequence = 2, current_status = INCOMING_AT
current_stop_sequence = 2, current_status = STOPPED_AT
current_stop_sequence = 3, current_status = IN_TRANSIT_TO // When vehicle departs stop 2
>
>If current_status is not provided, then current_stop_sequence should be updated based on the definition of IN_TRANSIT_TO, which means that the value only changes after the vehicle departs a stop.

**Comment by scmcca** [comment]
Pardon my lag. The vote ended on 2022-05-03 at 23:59:59 UTC. With only one +1 and outstanding feedback, the vote is inconclusive. I will rework the proposal based on the feedback above. Thanks!

**Pull request closed without merging**